sinatra-contrib/lib/sinatra/config_file.rb

```diff
@@ -149,24 +139,32 @@ def message
 
     private
 
-    # Given a +hash+ with some application configuration it returns the
-    # settings applicable to the current environment.  Note that this can only
-    # be done when all the keys of +hash+ are environment names included in the
-    # +environments+ setting (which is an Array of Strings).  Also, the
-    # returned config is a indifferently accessible Hash, which means that you
-    # can get its values using Strings or Symbols as keys.
-    def config_for_env(hash)
-      if hash.respond_to? :keys and hash.keys.all? { |k| environments.include? k.to_s }
-        hash = hash[environment.to_s] || hash[environment.to_sym]
-      end
+    # Given a +hash+ with some application configuration, returns the settings
```

Comment:
This word, `+hash+` refers to the new-renamed parameter (now called `config`), so the API docs generation will miss this annotation.